cleanup-renku-ci-deployments/entrypoint.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
+
+if test -z "$RENKUBOT_KUBECONFIG" ; then
+    echo 'Please specify a kubeconfig that can be used for the helm and kubectl commands.'
+    exit 1
+fi
+
+if test -z "$GITLAB_TOKEN" ; then
+    echo 'Please specify a GITLAB TOKEN.'
+    exit 1
+fi
+
+KUBECONFIG=${KUBECONFIG:-"/.kubeconfig"}
+HELM_CI_RELEASE_REGEX=".+-ci-.+|^ci-.+"
+HELM_RELEASE_REGEX="${HELM_RELEASE_REGEX:=".*"}"
+K8S_CI_NAMESPACE_REGEX=".+-ci-.+|^ci-.+"
+MAX_AGE_SECONDS=${MAX_AGE_SECONDS:=604800}
+GITLAB_URL="https://dev.renku.ch/gitlab"
+
+echo "$RENKUBOT_KUBECONFIG" > "$KUBECONFIG" && chmod 400 "$KUBECONFIG"
+echo "Looking for CI releases with regex $HELM_RELEASE_REGEX."
+echo "Looking in namespaces with regex $K8S_CI_NAMESPACE_REGEX."
+echo "Age threshold for deletion is $MAX_AGE_SECONDS seconds."
+
+# get a list of all applicable namespaces
+NAMESPACES=$(kubectl get namespaces -o json | jq -r ".items | map(.metadata.name | select(test(\"$K8S_CI_NAMESPACE_REGEX\"))) | .[]")
+NOW=$(date +%s)
+for NAMESPACE in $NAMESPACES
```

Comment:
you don't need to loop through namespaces. Instead, you can do something like

```
helm list --all-namespaces -d -f $HELM_CI_RELEASE_REGEX
```

and if a `HELM_RELEASE_REGEX` is specified you just need to deal with one specific helm release, in which case `HELM_CI_RELEASE_REGEX` becomes `HELM_RELEASE_REGEX`

Reply:
@rokroskar I know what you mean. But the reason I loop through namespaces is because I did not want to assume that the namespace is always the same as the renku release name. I know this is the case for now. But when and/or if we change this logic this actions will still work. Is that what you are referring to?

If I cannot assume that the helm release name equals the namespace then I think this is the easiest way to do this.

But maybe I misunderstand what you meant.

Reply:
If you fetch the json for the helm release I believe it includes the namespace so you can just grab it from there